src/store/type.ts

```diff
@@ -716,24 +719,29 @@ export type AudioPlayerStoreTypes = {
  * Singing Store Types
  */
 
-export type Tempo = {
-  position: number;
-  bpm: number;
-};
-
-export type TimeSignature = {
-  measureNumber: number;
-  beats: number;
-  beatType: number;
-};
-
-export type Note = {
-  id: string;
-  position: number;
-  duration: number;
-  noteNumber: number;
-  lyric: string;
-};
+// schemaはproject向けのものだが、そこから型定義を生成できるので一旦ここに置いておく
+// TODO: schemaをいい感じにおけるファイルを作る
```

Comment:
トーク周りの型はtype/preload.tsに置かれてたはずなのでそっちに揃えても良さそう？

Reply:
まだdomainディレクトリが無いですが、これができたら`src/domain/projectSchema.ts`辺り作ると良さそうな気がしました！

メモ程度ですが↓に書き足しておきました。

- https://github.com/VOICEVOX/voicevox/issues/1760

ちなみに指示語をコメントに書くのは正確性が落ちるので避けたほうが良かったりします。
それも合わせると、例えばこう？

```suggestion
// schemaはプロジェクトファイル用
// TODO: schemaをsrc/domain/projectSchema.tsに移動する
```

Reply:
ありがとうございます...!
suggestいただいたものが良さそうなので、そのまま使いたいと思います...!